services/db_shell/fixtures/test_data.cypher

```diff
@@ -15,6 +17,12 @@ MATCH (i:Identity{name: "test_export_key"})
 MATCH (i:Identity{name: "test_export_key"})
     MATCH (r:Resource{path: "/measurement"})
     CREATE (i)-[:Can{method: "POST"}]->(r);
+MATCH (i:Identity{name: "test_export_key"})
+    MATCH (r:Resource{path: "/survey"})
+    CREATE (i)-[:Can{method: "POST"}]->(r);
+MATCH (i:Identity{name: "test_export_key"})
+    MATCH (r:Resource{path: "/"})
+    CREATE (i)-[:Can{method: "GET"}]->(r);
```

Comment:
Why do we need to allow GET for export?

Reply:
Oh sorry this was to test some things about the apikey, (mostly though Postman). It's just doing GET on the homepage, the `http://localhost:PORT/` page.